nilearn/surface/surface.py

```diff
@@ -740,24 +746,28 @@ def load_surf_mesh(surf_mesh):
                               'array containing  the indices (into coords) of '
                               'the mesh faces. The input was a list with '
                               '%r elements.') % len(surf_mesh))
+    elif (hasattr(surf_mesh, "faces") & hasattr(surf_mesh,"coordinates")):
```

Comment:
```suggestion
    elif (hasattr(surf_mesh, "faces") and hasattr(surf_mesh,"coordinates")):
```
`&` in python is bitwise and; logical and is `and`